### PR TITLE
Update to electron 1.6.x

### PR DESCRIPTION
**Pull request by bpasero**
Refs: https://github.com/Microsoft/vscode/issues/23159

**Frameworks**
* V8 (53 => 56)
* Chrome (53 => 56)
* node.js (6.5.0 => 7.4.0 [breaking changes](https://github.com/nodejs/node/wiki/Breaking-changes-between-v6-and-v7))
* libuv (1.9.1 => 1.10.1)

### CODE REVIEW

File: src/vs/base/parts/tree/browser/treeView.ts [hunk 1]
```diff
@@ -488,6 +488,7 @@ export class TreeView extends HeightMap {
 		this.viewListeners.push(DOM.addDisposableListener(this.domNode, 'mousedown', (e) => this.onMouseDown(e)));
 		this.viewListeners.push(DOM.addDisposableListener(this.domNode, 'mouseup', (e) => this.onMouseUp(e)));
 		this.viewListeners.push(DOM.addDisposableListener(this.wrapper, 'click', (e) => this.onClick(e)));
+		this.viewListeners.push(DOM.addDisposableListener(this.wrapper, 'auxclick', (e) => this.onClick(e))); // >= Chrome 56
```
Comment (auchenberg) [inline src/vs/base/parts/tree/browser/treeView.ts]: If AUXCLICK is getting disabled, why add an event listener for the event type?

Comment (bpasero) [inline src/vs/base/parts/tree/browser/treeView.ts]: @auchenberg our code is not just running in VS Code, see also https://github.com/Microsoft/monaco-editor

Comment (auchenberg) [inline src/vs/base/parts/tree/browser/treeView.ts]: Ah, makes sense. Just found it puzzling.

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/pull/23842

**Pull request closed without merging**